src/lib/coda_base/transaction_witness.ml

```diff
@@ -1,21 +1,15 @@
-open Module_version
-
 module Stable = struct
   module V1 = struct
     module T = struct
-      let version = 1
-
-      let __versioned__ = true
-
-      (* TODO : version Sparse_ledger *)
-      type t = {ledger: Sparse_ledger.t} [@@deriving sexp, bin_io]
+      type t = {ledger: Sparse_ledger.Stable.V1.t}
+      [@@deriving sexp, bin_io, version]
     end
 
     include T
-    include Registration.Make_latest_version (T)
```

Comment:
The registration isn't required, but it doesn't hurt, and could be useful later. I think it should be left in, because it helps verify that `T` has all the version-y bits.